wgsl/index.bs

```diff
@@ -7679,9 +7689,14 @@ format to channel format.
 
 **Note:**
 
-If any of the parameters are out of bounds, then the call to `textureStore()`
-does nothing.
+An out-of-bounds access occurs if:
+* any element of `coords` is outside the range `[0, textureDimensions(t))`
+    for the corresponding element
+* `array_index` is outside the range of `[0, textureNumLayers(t))`
 
+If an out-of-bounds access occurs, the built-in function may do any of the following:
+* not be executed
+* store `value` to some in bounds texel
```

Comment:
nit: you've used `out-of-bounds` and `within bounds` above, so `within bounds` or `in-bounds` ?

Reply:
Thanks, I went and checked my previous PR and it did not hyphenate out of bounds, so I've removed the hyphenation here.